bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcView.kt

```diff
@@ -111,6 +117,24 @@ internal class BcmcView @JvmOverloads constructor(
         }
     }
 
+    private fun initCardHolderInput() {
+        binding.textInputLayoutCardHolder.isVisible = delegate.configuration.isHolderNameRequired
+        binding.editTextCardHolder.setOnChangeListener {
+            delegate.updateInputData { cardHolderName = binding.editTextCardHolder.rawValue }
+            binding.textInputLayoutCardHolder.hideError()
+        }
+
+        binding.editTextCardHolder.onFocusChangeListener = OnFocusChangeListener { _, hasFocus ->
+            val cardHolderValidation = delegate.outputData.cardHolderNameField.validation
+            if (hasFocus) {
+                binding.textInputLayoutCardHolder.hideError()
+            } else if (!cardHolderValidation.isValid()) {
```

Comment:
we can use `cardHolderValidation is Validation.Invalid` so we won't have to cast it below. there are examples in `CardView`. I think this PR is a good place to refactor all the other places that uses `isValid()` for this purpose.

Reply:
Good one! I indeed got this from `CardView`, so will change it everywhere I can find